### ISSUE TITLE
Should there be Microsoft.Extensions.SecretManager.Tools package for older versions of .NET?

### ISSUE THREAD

**Issue by hlubovac** [aspnet/DotNetTools#198]
I realize the development of this is still in works, but I hope there will be matching package for older versions of .NET integrated into the dotnetcore template/setup (project.json > frameworks) - because the current one definitely does not pass the nuget-restore step.

Thanks.

**Comment by natemcmaster** [comment]
The tool should be installed in the "tools" section of project.json. This should be independent of the frameworks you are using in your project. If you're trying to install via NuGet CLI or GUI, it's probably being put in "dependencies", which is https://github.com/aspnet/DotNetTools/issues/157 and will be resolved in the next release.

**Comment by hlubovac** [comment]
I tried that, and it fails. What could I be missing? Here's what I think is relevant:

This. in `Tools` section: `"Microsoft.Extensions.SecretManager.Tools": "1.0.0-preview2-final"` results in this error [dotnet/nuget restore]:

"Package Microsoft.Extensions.SecretManager.Tools 1.0.0-preview2-final is not compatible with net461 (.NETFramework,Version=v4.6.1). Package Microsoft.Extensions.SecretManager.Tools 1.0.0-preview2-final supports: netcoreapp1.0 (.NETCoreApp,Version=v1.0)
    One or more packages are incompatible with .NETFramework,Version=v4.6.1."

Same problem with: `"Microsoft.Extensions.SecretManager.Tools": "*"`.

This won't work at all: `"Microsoft.Extensions.SecretManager.Tools": ""`.


Re: `frameworks` section:

     "net461": {
      "dependencies": {
      },
      "frameworkAssemblies": {
        "System.ServiceProcess": ""
      }
    }

No "netcoreapp1.0" in `frameworks` and no "Microsoft.NETCore.App" in `dependencies`. Having both the framework and the platform `:-)` just wouldn't work for me (nuget-restore problems), so I decided to degrade for the time being, as, at the moment, I needed something that I wasn't able to find in dotnetcore.
